iwyu_include_picker.cc

```diff
@@ -347,6 +347,7 @@ const IncludeMapEntry libc_include_map[] = {
   { "<bits/sigcontext.h>", kPrivate, "<signal.h>", kPublic },
   { "<bits/siginfo.h>", kPrivate, "<signal.h>", kPublic },
   { "<bits/signum.h>", kPrivate, "<signal.h>", kPublic },
+  { "<bits/signum-arch.h>", kPrivate, "<signal.h>", kPublic },
```

Comment:
Can you make the same change in `gcc.libc.imp`, please?

Reply:
Thanks!  Didn't know of that file.

Reply:
Done.